hbase-server/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupsBase.java

```diff
@@ -159,7 +159,7 @@ public void tearDownAfterMethod() throws Exception {
     int missing = NUM_SLAVES_BASE - getNumServers();
     LOG.info("Restoring servers: " + missing);
     for (int i = 0; i < missing; i++) {
-      ((MiniHBaseCluster) CLUSTER).startRegionServer();
+      ((MiniHBaseCluster) CLUSTER).startRegionServerAndWait(2000);
```

Comment:
I believe there was a test for this in the first version of the patch thats now been deleted. Do we need one for this patch?

Reply:
I think the existing tests cover the change (seeing as we had many test failures resulting from the change which had to be fixed)